### DISCUSSION TITLE
Where is pkg/go.mod used?

### DISCUSSION THREAD

**Discussion by mikeolubode** [milvus-io/milvus#31048] [Q&A and General discussion] [answered]
I am deploying Milvus cluster using the helmchart repo https://github.com/zilliztech/milvus-helm/tree/master.
I want to know if pkg/go.mod is used anywhere in building the docker image so I can modify some of the package versions. Thanks.

**Comment by yhmo** [comment, marked as answer]
Do you mean this file? https://github.com/milvus-io/milvus/blob/master/pkg/go.mod
It is to define the third-party versions for compiling milvus. I thing it has nothing to do with milvus-helm.
